src/runtime/runtime_internal.h

```diff
@@ -86,6 +87,17 @@ WEAK char *halide_int64_to_string(char *dst, char *end, int64_t arg, int digits)
 WEAK char *halide_uint64_to_string(char *dst, char *end, uint64_t arg, int digits);
 WEAK char *halide_pointer_to_string(char *dst, char *end, const void *arg);
 
+// Search the current process for a symbol with the given name.
+WEAK void *halide_get_symbol(const char *name);
+// Platform specific implementations of dlopen/dlsym/dlclose.
+WEAK void *halide_load_library(const char *name);
+WEAK void *halide_get_library_symbol(void *lib, const char *name);
+WEAK void halide_free_library(void *lib);
```

Comment:
Good point, I'll just remove it for now. BTW, it's named free_library out of habit: on windows, the functions are LoadLibrary and FreeLibrary.